src/generator/src/commonMain/kotlin/community/flock/wirespec/generator/Generator.kt

```diff
@@ -0,0 +1,109 @@
+import community.flock.kotlinx.rgxgen.RgxGen
+import community.flock.wirespec.compiler.core.parse.AST
+import community.flock.wirespec.compiler.core.parse.Definition
+import community.flock.wirespec.compiler.core.parse.Endpoint
+import community.flock.wirespec.compiler.core.parse.Enum
+import community.flock.wirespec.compiler.core.parse.Refined
+import community.flock.wirespec.compiler.core.parse.Type
+import community.flock.wirespec.compiler.core.parse.Type.Shape.Field.Reference
+import community.flock.wirespec.compiler.core.parse.Union
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonNull
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlin.random.Random
+
+fun AST.generate(type: String, random: Random = Random.Default) = Generator.generate(this, type, random)
```

Comment:
Why is this almost the same as the function on line 21? That line could also be an extension function

Reply:
All extention functions